### PR TITLE
Changes 2.2.8

### PR DESCRIPTION
**Pull request by singhravidutt**

### CODE REVIEW

File: gcs/CHANGES.md [hunk 1]
```diff
@@ -21,6 +21,10 @@
 
 1. Increasing gRPC read timeout from 30 seconds to 3600 seconds.
 
+1. Updating max list items per call from 1024 -> 5000. (#872)
```
Comment (arunkumarchacko) [inline gcs/CHANGES.md:24]: please follow the pattern and change "->" to "to"?

File: gcs/CHANGES.md [hunk 2]
```diff
@@ -21,6 +21,10 @@
...
+
+1. Adding support for invocationId in Json path. (#873)
```
Comment (arunkumarchacko) [inline gcs/CHANGES.md:26]: Remove the PR number from the description.

File: gcs/CHANGES.md (hunk 1 above)
Comment (singhravidutt) [reply gcs/CHANGES.md]: addressed.

File: gcs/CHANGES.md (hunk 2 above)
Comment (singhravidutt) [reply gcs/CHANGES.md]: addressed.

**Review by arunkumarchacko — APPROVED** [review]

**Pull request merged**